boards/laird_connect/bl654_dvk/doc/bl654_dvk.rst

```diff
@@ -9,6 +9,9 @@ Overview
 The BL654 Development Kit hardware provides
 support for the Laird Connectivity BL654 module powered by a Nordic Semiconductor nRF52840 ARM Cortex-M4F CPU.
 
+It is also pin compatible with the BL654PA which adds a power amplifier. The "pa" variant provides
+this compatibility. Use board `bl654_dvk/nrf52840/pa` to build for that target.
```

Comment:
you for to add extra \`, as pointed out earlier.
```suggestion
this compatibility. Use board ``bl654_dvk/nrf52840/pa`` to build for that target.
```

Reply:
I missed this. Updated.